### PR TITLE
How to install and setup VMware vSphere

### PR DESCRIPTION
**Pull request by asciiprod**
I have started to port/rewrite my wiki article on installing VMware vSphere. It though it would be a good start since it is one of the most viewed ones. This is still in incomplete as the text needs some restructuring and removal of dated information.

### CODE REVIEW

**Comment by MasterWayZ** [comment]
Hello, I see that this pull request hasn't been updated in a while. I wanted to create a guide of installing VMware ESXi, followed by guides of deploying vCenter and NSX. However, since this guide covers ESXi part, I am not sure what to do. I wanted to both make a written guide as well as video guide in the subjects.

My question is, what should I do? Just create the guides on the latter two subjects, make nothing, just make a video guide on ESXi... What should I do?

**Comment by n-se** [comment]
Closing in favor of #118

**Comment by asciiprod** [comment]
#118 is about vCenter not about vSphere.

File: tutorials/install-and-configure-routervm-for-vmware-vsphere/01.en.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+### Step 2.1 - Creation of Router VM
+
+In this tutorial, Debian 10 Buster will be used as distribution for the Router VM. 
+
+Go to `Navigator -> Virtual Machines` and create a new VM. Select `Create a new virtual machine` as creation type and go to the next step. Name the machine e.g. `RouterVM` and select `Linux` as Guest OS family and `Debian GNU/Linux 10 (64-bit)` as Guest OS version. Select your datastore and go to the next step. On step 4, select "Bridged Network" on the first network adapter and create a second network adapter and select `Routed Network` as network on it. On the first network adapter, do not forget to set the MAC address of the additional IPv4 address. 
+
+![](images/config-routervm.png)
+
+When virtual machine is created, proceed with the installation of Debian 10.
```
Comment (LKaemmerling) [inline tutorials/install-and-configure-routervm-for-vmware-vsphere/01.en.md]:
```suggestion
When the virtual machine was created, proceed with the installation of Debian 10.

File: tutorials/install-and-configure-routervm-for-vmware-vsphere/01.en.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+The IPv6 subnet is routed to the default link-local address (which is derived from the MAC address) of the main IP. Via [Hetzner Robot](https://robot.your-server.de) the routing of the IPv6 subnet can be switched to the link-local address of the virtual MAC (in other words, the additional single IP). This can be done in [Hetzner Robot](https://robot.your-server.de), using the same symbol which is found next to additional single IPs to request virtual MAC addresses. In this case, the host running vSphere receives no IPv6 address. This is neither necessary not possible because ESXi does not work with an `fe80::1` gateway.
+
+## Step 1 - Creation of Port groups and vSwitch
+
+Go to `Navigator -> Networking -> Virtual switches ` and create a new virtual switch without any uplink.
+
+![vSwitch config](images/config-vswitch.png "vSwitch config")
+
+Go to ` Port groups` and rename the port group `VM Network` to something else (e.g. `Bridged Network`) to avoid confusion. Create a new Port Group and name it e.g. `Routed Network` and use the vSwitch `VM Switch`.
```
Comment (LKaemmerling) [inline tutorials/install-and-configure-routervm-for-vmware-vsphere/01.en.md]: Space to much (Port groups)

File: tutorials/install-and-configure-routervm-for-vmware-vsphere/01.en.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+It is based on VMware ESXi 6.7 Update 2 (Build 13006603) without any vCenter Server.
+
+## Requirements
+
+- an additional IPv4 address
+- a separate MAC address for the additional IPv4 address (can be generated in Robot)
+- a subnet (which gets routed over the additional IPv4 address)
+- VMware vSphere Web Client
+
+Using a subnet requires at least one additional single IP address which must be setup with a virtual machine that will function as a router for the subnet. When ordering a subnet, **please make sure to add a note to request it should be routed to the additional single IP address.**
```
Comment (LKaemmerling) [inline tutorials/install-and-configure-routervm-for-vmware-vsphere/01.en.md]:
```suggestion
Using a subnet requires at least one additional single IP address which must be set up with a virtual machine that will function as a router for the subnet. When ordering a subnet, **please make sure to add a note to request it should be routed to the additional single IP address.**
```

The first sentence sounds a bit weird

File: tutorials/install-and-configure-routervm-for-vmware-vsphere/01.en.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
+---
+path: "/tutorials/install-and-configure-routervm-for-vmware-vsphere"
+slug: "install-and-configure-routervm-for-vmware-vsphere"
+date: "2019-03-08"
+title: "Install and configure a Router VM for VMware vSphere"
+short_description: ""
```
Comment (n-se) [inline tutorials/install-and-configure-routervm-for-vmware-vsphere/01.en.md]: Description is needed for the tutorial

File: tutorials/install-and-configure-routervm-for-vmware-vsphere/01.en.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+tags: ["Hetzner Official", "vmware", "vsphere"]
+author: "Hetzner Online"
+author_link: "https://github.com/hetzneronline"
+author_img: "https://avatars3.githubusercontent.com/u/30047064?s=200&v=4"
```
Comment (n-se) [inline tutorials/install-and-configure-routervm-for-vmware-vsphere/01.en.md]:
```suggestion
author_img: "https://avatars3.githubusercontent.com/u/30047064"
```

File: tutorials/install-and-configure-routervm-for-vmware-vsphere/01.en.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+For the test Client VM, Debian 10 will be used as distribution.
+
+**Do not forget to use the `Routed Network` Port group for the Client VM**.
+
+Edit the file `/etc/network/interfaces` and add the following lines (interface names may need to get changed if there are other):
+
+```
+auto ens160
+  address 10.0.0.2
```
Comment (n-se) [inline tutorials/install-and-configure-routervm-for-vmware-vsphere/01.en.md]: I think `iface ens160 inet static` needs to be added here?

**Pull request merged**